src/v2/cookbook/serverless-blog.md

```diff
@@ -0,0 +1,289 @@
+    </router-link>
+    <router-link v-if="post.meta.next_post" :to="/blog/ + post.meta.next_post.slug" class="button">
+      {{ post.meta.next_post.title }}
+    </router-link>
+  </div>
+</template>
+```
+
+Here's a preview:
+
+![buttercms-blogdetail](https://user-images.githubusercontent.com/160873/36868506-218c86b6-1d5e-11e8-8691-0409d91366d6.png)
+
+
+Now our app is pulling all blog posts and we can navigate to individual posts. However, our next/previous post buttons are not working.
+
+One thing to note when using routes with params is that when the user navigates from /blog/foo to /blog/bar, the same component instance will be reused. Since both routes render the same component, this is more efficient than destroying the old instance and then creating a new one. 
+
+<p class="tip">Be aware, that using the component this way will mean that the lifecycle hooks of the component will not be called. Visit the Vue.js docs to learn more about [Dynamic Route Matching](https://router.vuejs.org/en/essentials/dynamic-matching.html)</p>
+
+To fix this we need to simply watch the `$route` object and call `getPost()` when the route changes.
+
+Updated `script` section in `components/BlogPost.vue`:
+
+```javascript
+<script>
+  import { butter } from '@/buttercms'
+  export default {
+    name: 'blog-post',
+    data() {
+      return {
+        post: {}
+      }
+    },
+    methods: {
+      getPost() {
+        butter.post.retrieve(this.$route.params.slug)
+          .then((res) => {
+            // console.log(res.data)
+            this.post = res.data
+          }).catch((res) => {
+            console.log(res)
+          })
+      }
+    },
+    watch: {
+      $route(to, from) {
+        this.getPost()
+      }
+    },
+    created() {
+      this.getPost()
+    }
+  }
+</script>
+```
+
+Now your app has a working blog that can be updated easily in the ButterCMS dashboard.
+
+## Categories, Tags, and Authors
+
+Use Butter's APIs for categories, tags, and authors to feature and filter content on your blog.
+
+See the ButterCMS API reference for more information about these objects:
+
+* [Categories](https://buttercms.com/docs/api/?ruby#categories)
+* [Tags](https://buttercms.com/docs/api/?ruby#tags)
+* [Authors](https://buttercms.com/docs/api/?ruby#authors)
+
+Here's an example of listing all categories and getting posts by category. Call these methods on the `created()` lifecycle hook:
+
+```
+methods: {
+  ...
+  getCategories() {
+    butter.category.list()
+      .then((res) => {
+        console.log('List of Categories:')
+        console.log(res.data.data)
+      })
+  },
+  getPostsByCategory() {
+    butter.category.retrieve('example-category', {
+        include: 'recent_posts'
+      })
+      .then((res) => {
+        console.log('Posts with specific category:')
+        console.log(res)
+      })
+  }
+},
+created() {
+  ...
+  this.getCategories()
+  this.getPostsByCategory()
+}
+```
+## Wrap up
```

Comment:
Please put the wrap up section after the alternative patterns section. I don't think you need the part about the feedback and PR, because that's kind of self explanatory, but I could go either way on it :) It's not a bad thing to include.